source/test/F0.CodeAnalysis.Testing/F0.CodeAnalysis.Testing.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.3.1" />
+    <PackageReference Include="xunit.assert" Version="2.4.2" />
```

Comment:
We already consumed `xunit.assert` as a transitive dependency.
Updated the package to a version that has nullable annotations and the respective nullable attributes.